sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -75,10 +79,46 @@ export interface DecryptResult {
 }
 
 // @public
-export interface DeletedKey extends Key {
-    readonly deletedDate?: Date;
-    readonly recoveryId?: string;
-    readonly scheduledPurgeDate?: Date;
+export interface DeletedKey {
+    keyMaterial?: JsonWebKey;
+    properties: KeyProperties & {
+        readonly recoveryId?: string;
+        readonly scheduledPurgeDate?: Date;
+        readonly deletedDate?: Date;
+    };
+}
+
+// @public
+export class DeleteKeyPoller extends Poller<DeleteKeyPollOperationState, DeletedKey> {
+    constructor(options: DeleteKeyPollerOptions);
+    delay(): Promise<void>;
+    intervalInMs: number;
+}
+
+// @public (undocumented)
+export interface DeleteKeyPollerOptions {
+    // Warning: (ae-forgotten-export) The symbol "KeyClientInterface" needs to be exported by the entry point index.d.ts
```

Comment:
You still have some unexported symbols

Reply:
I didn't intend DeleteKeyPollerOptions to be exported. I'll un-export it.